activiti-engine/src/test/java/org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.java

```diff
@@ -54,14 +54,26 @@ public void testThrowErrorWithEmptyErrorCode() {
     }
   }
 
-  @Deployment
+  public void testThrowErrorWithoutErrorCode() {
+    try {
+      repositoryService.createDeployment().addClasspathResource("org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.testThrowErrorWithoutErrorCode.bpmn20.xml").deploy();
+      fail("ActivitiException expected");
```

Comment:
I know that this class is using this pattern to test exceptions, but do you mind to migrate it to `catchThrowable` from `AssertJ`. It's more readable in my option. You can find an example here https://github.com/Activiti/activiti-cloud-query-service/blob/develop/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/events/handlers/TaskEntityCandidateGroupAddedEventHandlerTest.java#L80-L86